lib/video_info/providers/youtube_scraper.rb

```diff
@@ -64,7 +64,11 @@ def meta_nodes
 
       def meta_node_value(name)
         if available?
-          node = meta_nodes.detect { |n| n.attr('name').value == name }
+          node = meta_nodes.detect do |n|
+            n.attr('name').present? && n.attr('name').value == name
```

Comment:
`.present?` is not a Ruby method and is only supported when `activesupport` gem is present, which is not a dependency of this gem, therefore we cannot using it here.

I would write it that way:
``` ruby
n.attr('name') && n.attr('name').value == name
```